### ISSUE TITLE
[Question] Removing padding

### ISSUE THREAD

**Issue by halaszbalazs** [indiespirit/react-native-chart-kit#148]
Hey,

First of all thanks for this great component - really useful and straightforward.
However there is something that I don't know how to achieve with LineChart. It renders a padding in the view on the left and bottom side (see screenshot below - padding marked with red overlay). Is there any way to remove this padding? Thanks!

![screenshot](https://user-images.githubusercontent.com/12443792/63775094-f0d07900-c8de-11e9-84ff-6d926f09adca.png)

**Comment by halaszbalazs** [comment]
Thank you @Hermanya I'll look into it in the near future then!

**Comment by JaredDahlke** [comment]
has anyone figured out how to center the graph , given that the left margin is not changeable?

**Comment by rodriguezmanu** [comment]
+1

**Comment by igorgomeslima** [comment]
@Hermanya this feature is now changeble?

**Comment by Hermanya** [comment]
@igorgomeslima that PR was never merged

**Comment by igorgomeslima** [comment]
Yeah @Hermanya , I see this on your comment [Here](https://github.com/indiespirit/react-native-chart-kit/pull/174#issuecomment-539786206)...


I'm trying to set up a configuration to remove the "additional paddings" and "round only `borderBottomRight`" of LineChart, but without success...

Is that supported?

![image](https://user-images.githubusercontent.com/2649181/78295679-e4f70600-7502-11ea-897d-fd6c0f7de329.png)

With code:

```
<View>
      <LineChart
        data={{
          datasets: [
            {
              data: [/*...*/]
            }
          ]
        }}
        width={300} // from react-native
        height={300}
        yAxisLabel=""
        yAxisSuffix=""
        yAxisInterval={1} // optional, defaults to 1
        chartConfig={{
          backgroundColor: "#e26a00",
          backgroundGradientFrom: "#fb8c00",
          backgroundGradientTo: "#ffa726",
          propsForLabels: {
            fontSize: "11"
          },
          decimalPlaces: 2, // optional, defaults to 2dp
          color: (opacity = 1) => `rgba(255, 255, 255, ${opacity})`,
          labelColor: (opacity = 1) => `rgba(255, 255, 255, ${opacity})`,
          style: {
            borderRadius: 16
          },
          propsForDots: {
            r: "6",
            strokeWidth: "2",
            stroke: "#ffa726"
          }
        }}
        withVerticalLabels={false}
        withHorizontalLabelsLabels={false}
        withInnerLines={false}
        withOuterLines={false}
        bezier
        style={{
          borderBottomRightRadius: 15,
          paddingRight:0
        }}
      />
    </View>
```

**Comment by RominHalltari** [comment]
+1

**Comment by tayloraleach** [comment]
+1

**Comment by auriga-dev** [comment]
style={{
      paddingRight:0,
    }}

will remove the right space of the chart so that if you want to hide the labels at will start at the left of the screen.

**Comment by ruiaraujolink** [comment]
I have the same issue, and this snack code has the same in lineChart [snack code](https://snack.expo.io/@aikismax/react-native-chart-kit), right margin is too big, and paddingRight: 0 does not remove it

**Comment by ManuViola77** [comment]
The same happens to me, with `paddingRight: 0` I ironically remove the left padding, but I still have a rightPadding of size 16. My only solution is to set the chart's width as `SCREEN_WIDTH + 16` but it seems there should be a better solution

**Comment by DanielWFrancis** [comment]
@ManuViola77's solution worked but it feels gross sending that out into production 😂

**Comment by rafaelaugustos** [comment]
Any new solution ?

**Comment by 99Jannis99** [comment]
Need a solution to?

**Comment by SakeFabi** [comment]
There is still no solution to this one, is there? The charts look generally cool, but like this it is very hard to style.

**Comment by mahirsodagar** [comment]
> Yeah @Hermanya, I see this on your comment [Here](https://github.com/indiespirit/react-native-chart-kit/pull/174#issuecomment-539786206)...
> 
> I'm trying to set up a configuration to remove the "additional paddings" and "round only `borderBottomRight`" of LineChart, but without success...
> 
> Is that supported?
> 
> ![image](https://user-images.githubusercontent.com/2649181/78296276-ccd3b680-7503-11ea-9fa4-d83bbd6090aa.png)
> 
> With code:
> 
> ```
> <View>
>       <LineChart
>         data={{
>           datasets: [
>             {
>               data: [/*...*/]
>             }
>           ]
>         }}
>         width={300} 
>         height={300}
>         yAxisLabel=""
>         yAxisSuffix=""
>         chartConfig={{
>           backgroundColor: "#e26a00",
>           backgroundGradientFrom: "#fb8c00",
>           backgroundGradientTo: "#ffa726",
>           propsForLabels: {
>             fontSize: "11"
>           },
>           decimalPlaces: 2, // optional, defaults to 2dp
>           color: (opacity = 1) => `rgba(255, 255, 255, ${opacity})`,
>           labelColor: (opacity = 1) => `rgba(255, 255, 255, ${opacity})`,
>           style: {
>             borderRadius: 16
>           },
>           propsForDots: {
>             r: "6",
>             strokeWidth: "2",
>             stroke: "#ffa726"
>           }
>         }}
>         withVerticalLabels={false}
>         withHorizontalLabelsLabels={false}
>         withInnerLines={false}
>         withOuterLines={false}
>         bezier
>         style={{
>           borderBottomRightRadius: 15,
>           paddingRight:0
>         }}
>       />
>     </View>
> ```

Try This

style={{
   height: 25, // half of height prop
   width: 85, // adujust to width
   paddingTop: 0,
   paddingRight: 0,
 }}
